CMakeLists.txt

```diff
@@ -148,16 +204,24 @@ if(NOT CUKE_DISABLE_FUNCTIONAL)
         add_executable(functional-steps EXCLUDE_FROM_ALL ${CUKE_DYNAMIC_CPP_STEPS})
         target_link_libraries(functional-steps ${CUKE_LIBRARIES})
 
-        # TODO It does not escape paths
-        set(CUKE_COMPILE_DYNAMIC_CPP_STEPS "${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR} --target functional-steps")
+        set(CUKE_STEPS_DIR "${CMAKE_BINARY_DIR}")
+        if(MSVC)
+            set(CUKE_STEPS_DIR "${CUKE_STEPS_DIR}/${CMAKE_CFG_INTDIR}")
+        endif()
 
+        set(CUKE_COMPILE_DYNAMIC_CPP_STEPS "cmake --build ../build --target functional-steps")
```

Comment:
This looks worse than the original line. `cmake` might not be in the path, `../build` might not be the build directory. Do we need the paths escaped for the Windows build?